### DISCUSSION TITLE
Using the Relation field as a prompt, not a selection field

### DISCUSSION THREAD

**Discussion by ega22a** [orchidsoftware/platform#2381] [Q&A] [answered]
Hi all, I need to use the Relation field as a field with a drop down list, but if there is no data in the drop down list, I need the data entered to be saved in the input field, not discarded if there is no selection from the model. I haven't found how to implement this yet, so I'm asking for help.

Thanks!

P.S. Here's an example of a field. And if there is no data, it resets it.
```php
use Orchid\Screen\Fields\Relation;

Relation::make('passportIssuer')
    -> title('Place of passport issuance')
    -> help('Enter the place where the passport was issued.')
    -> fromModel(Issuer::class, 'name')
    -> searchColumns('code'),
```
![Alt Text](https://github.com/ega22a/ws-ekb-2021/blob/master/%D0%92%D0%B8%D0%B4%D0%B5%D0%BE%20%D0%B1%D0%B5%D0%B7%20%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%20%E2%80%94%20%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D0%BD%D0%BE%20%D0%B2%20Clipchamp%20(1).gif?raw=true)

**Comment by tabuna** [comment, marked as answer]
`Relation` assumes a second model. For example, in `Select` you can add your own value like this:

```php
use Orchid\Screen\Fields\Select;

Select::make('passportIssuer')
    ->title('Place of passport issuance')
    ->allowAdd()
    //...

```

**Reply by ega22a** [reply]
I don't quite understand this method... The documentation doesn't say anything about it, and when I use this method, nothing changes for the Select field. I can't enter my own data...

```php
Select::make('passportIssuer')
    -> title('Место выдачи паспорта')
    -> help('Введите место выдачи паспорта')
    -> allowAdd()
    -> fromModel(Issuer::class, 'name'),
```

![Alt Text](https://github.com/ega22a/ws-ekb-2021/blob/master/%D0%92%D0%B8%D0%B4%D0%B5%D0%BE%20%D0%B1%D0%B5%D0%B7%20%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%20%E2%80%94%20%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D0%BD%D0%BE%20%D0%B2%20Clipchamp%20(1)%20(1).gif?raw=true)

**Reply by ega22a** [reply]
Oh, okay, I get it. I had the 13.1.0 dependency installed. Now I upgraded to 13.4.0, and it really does give me what I need. Thank you very much!

**Comment by The01bestMan** [comment]
bonjour mes devanciers svp je veux faire menu de catégoires et sous categoires  mes catégories s'affichent 
mais je n'arrive pas à afficher mes sous catégories svp j'ai besoin d'aide

**Reply by ega22a** [reply]
Can you provide an example of what you want? Well, and preferably write it in English.
